rules/xcodeproj.bzl

```diff
@@ -0,0 +1,263 @@
+load("@build_bazel_rules_apple//apple:providers.bzl", "AppleBundleInfo")
+load("@bazel_skylib//lib:paths.bzl", "paths")
+
+def _get_transitive(deps, provider, field):
+    return [
+        getattr(dep[provider], field)
+        for dep in deps
+        if dep and provider in dep
+    ]
+
+_TargetInfo = provider()
+_SrcsInfo = provider()
+
+def _dir(o):
+    return [
+        x
+        for x in dir(o)
+        if x not in ("to_json", "to_proto")
+    ]
+
+def _xcodeproj_aspect_impl(target, ctx):
+    providers = []
+
+    deps = []
+    deps += getattr(ctx.rule.attr, "deps", [])
+    deps += getattr(ctx.rule.attr, "infoplists", [])
+    deps.append(getattr(ctx.rule.attr, "entitlements", None))
+
+    # TODO: handle apple_resource_bundle targets
+    if AppleBundleInfo in target:
```

Comment:
@segiddins , do you think we should  resolve this TODO?

Reply:
I don’t think it’s a blocker before merging

Reply:
Agreed